### ISSUE TITLE
Python3 support

### ISSUE THREAD

**Issue by rodo** [rafaelma/pgbackman#50]
Hi,

Have you plan to port pgbackman to python 3 ?

**Comment by usit-gd** [comment]
Yes, it is in the TODO list and I have started looking at what we need to do. 
Thank you for your patch in #51 . I will merge it in a few days.
